src/components/sendV2/summary/summary.test.js

```diff
@@ -49,8 +49,11 @@ describe('Summary', () => {
   const props = {
     t: v => v,
     account: {
+      address: '12345L',
```

Comment:
can't use from the accounts constant file?

Reply:
made the change